Sources/TokamakCore/Views/View.swift

```diff
@@ -22,12 +22,23 @@ public protocol View {
 }
 
 public extension Never {
+  @_spi(TokamakCore)
   var body: Never {
```

Comment:
Is there a reason that the body implementation from the PrimitiveView conformance is not good enough?
There probably is, but I'm not at a computer, so I can't try it out right now. :-)

Reply:
It's just that a `Never` instance can, as the name suggests, never exist. Thus, I thought it would be misleading to use the primitive view conformance. Not to mention that relying on the `PrimitiveView` default implementation causes problems with `Never`-body scenes (for reasons I don't fully understand).

In my opinion, instance members of no-case enums, like `Never`, should non require a body anyway. This behavior is actually in place for functions taking no-case enums, meaning that `func a(_: Never) -> Int {}` is perfectly valid.